docs/pages/distribution/runtime-version.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Runtime Version
+---
+
+Expo Update allows you to publish over the air updates to apps. The update must be for an app running a
+[compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. You may use a runtime other than an Expo SDK, by specifying a runtime version.
```

Comment:
```suggestion
Over-the-air updates with `expo-updates` will only work in binaries with a [compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. By default, the runtime version is the Expo SDK version: the runtime is the set of libraries in the Expo SDK. Apps built with EAS Build will include only the dependencies that are present at the time that the app binary is built, and so the Expo SDK version will often not properly describe the runtime of the app. In these cases, you can start using `runtimeVersion` and it will take precedence over the Expo SDK version.
```